### ISSUE TITLE
event_trigger_type syntax in documentation doesn't work

### ISSUE THREAD

**Issue by dwmintz** [google-github-actions/deploy-cloud-functions#307]
### TL;DR

This [tutorial](https://cloud.google.com/blog/topics/developers-practitioners/deploying-serverless-platforms-github-actions) suggests that you use the `providers/cloud.storage/eventTypes/object.finalize` syntax for event triggers, but this leads to the extremely cryptic "The request has errors" error.

Using the `google.storage.object.finalize` syntax instead worked, but I only discovered that in this [issue](https://github.com/google-github-actions/deploy-cloud-functions/issues/58)

### Expected behavior

For the function to deploy, or at least provide a usable error.

### Observed behavior

```
status: {
code: 3
message: "The request has errors"
}
```

### Action YAML

```yaml
name: Deploy Google Cloud Function
on: push

env:
  jurisdiction: ***-dev
  creds: ${{ secrets.ENV_GOOGLE_APP_CREDS_FILE }} 


jobs:
  deploy:
    runs-on: ubuntu-latest
    environment: ****-dev

    steps:
      - name: Checkout code
        uses: actions/checkout@v2

      - id: 'auth'
        name: 'Authenticate to Google Cloud'
        uses: 'google-github-actions/auth@v0'
        with:
          credentials_json: '${{ env.creds }}'

      - name: Deploy Cloud Function
        id: deploy-background-function
        uses: google-github-actions/deploy-cloud-functions@v0.6.0
        with:
          name: ******
          runtime: python37
          event_trigger_type: providers/cloud.storage/eventTypes/object.finalize
          event_trigger_resource: projects/_/buckets/******.appspot.com
          entry_point: ******
```
```


### Log output

_No response_

### Additional information

_No response_

**Comment by sethvargo** [comment]
Hi @dwmintz - we've done a fairly significant refactor of this codebase, so you should get better error messages in the future from this GitHub Action. That being said, the error is coming back from the Cloud Functions API, which we do not have control over.

You can find the list of trigger event types in the Cloud Functions documentation. It's possible the trigger type changed after than blog post was published; I'll try to get it updated. In the meantime, the Cloud Functions documentation should be the source of truth here: https://cloud.google.com/functions/docs/calling and https://cloud.google.com/functions/docs/calling/storage
